examples/apptainer-rootful.yaml

```diff
@@ -19,21 +25,15 @@ containerd:
   system: false
   user: false
 provision:
-- mode: system
-  script: |
-    #!/bin/bash
-    set -eux -o pipefail
-    modprobe squashfs >/dev/null 2>&1 && exit 0
-    dnf install -y kernel-modules-$(uname -r)
 - mode: system
   script: |
     #!/bin/bash
     set -eux -o pipefail
     command -v apptainer >/dev/null 2>&1 && exit 0
-    dnf -y install apptainer apptainer-suid
-    # See https://fedoraproject.org/wiki/Features/tmp-on-tmpfs
-    echo "APPTAINER_TMPDIR=/var/tmp" > /etc/profile.d/apptainer.sh
```

Comment:
note: the ubuntu installation has a normal `/tmp` (not tmpfs), so doesn't need this override